docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc

```diff
@@ -147,7 +147,7 @@ person.status = Status.Alive;
 person.persist();
 
 // note that once persisted, you don't need to explicitly save your entity: all
-// modifications are automatically persisted on transaction commit.
+// modifications are automatically persiion commit.
```

Comment:
This must be unintentional ;)

Reply:
Yes, I should have seen it before pushing, sorry, corrected